content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,346 @@
+> * [Kubernetes Response Engine, Part 5 : Falcosidekick + Argo]({{< ref "/blog/falcosidekick-reponse-engine-part-5-argo" >}})
+> * [Kubernetes Response Engine, Part 6 : Falcosidekick + Cloud Run]({{< ref "/blog/falcosidekick-reponse-engine-part-6-cloud-run" >}})
+> * [Kubernetes Response Engine, Part 7 : Falcosidekick + Cloud Functions]({{< ref "/blog/falcosidekick-reponse-engine-part-7-cloud-functions" >}})
+
+----
+
+Today, we will set up another KRE (Kubernetes Response Engine) based on Flux v2. If you don't know about Flux v2, let me explain it in a few words. Flux is a tool for keeping Kubernetes clusters in sync with configuration sources (such as Git repositories) and automating updates to the configuration when new code is available to deploy.
+
+To learn more about Flux v2, see [Flux Documentation](https://fluxcd.io/docs/).
+
+Flux v2 might look like a GitOps tool. It is, in reality, another GitOps tool in that it watches Github repositories for configuration changes and keeps the current state and the desired state always in sync. It does that on top of Kubernetes by using a bunch of CRs (Custom Resources). However, unlike Knative Eventing, Tekton Triggers, and Argo Events, Flux v2 does not support an eventing system to forward events from one point to another.
+
+To set up KRE with Flux v2, we will create a small project that listens to events and updates the GitHub repository, which Flux v2 monitors to alter the desired state. For example, get the event of a pwned pod, then change its replicas to zero within the deployment YAML file.
+
+The reference architecture given below illustrates the content of this blog.
+
+![kre_flux_v2_gitops_toolkit](/docs/images/kre_flux_v2_gitops_toolkit.png)
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Prerequisites](#prerequisites)
+- [Tutorial](#tutorial)
+  - [Provision local Kubernetes Cluster](#provision-local-kubernetes-cluster)
+  - [Install Flux V2 - GitOps Toolkit](#install-flux-v2---gitops-toolkit)
+  - [Install falco-event-listener](#install-falco-event-listener)
+  - [Installing Falco and Falcosidekick](#installing-falco-and-falcosidekick)
+- [Test](#test)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Prerequisites
+
+* minikube v1.21.0
+* helm v3.6.2+gee407bd
+* kubectl v1.21.1
+* ko v0.8.3
+* flux v0.16.0
+* gcloud v347.0.0
+
+## Tutorial
+
+### Provision local Kubernetes Cluster
+
+```shell
+$ minikube config view
+- cpus: 3
+- driver: virtualbox
+- memory: 8192
+
+minikube start
+```
+
+### Install Flux V2 - GitOps Toolkit
+
+I highly recommended that you check out [getting started](https://fluxcd.io/docs/get-started/) page of `Flux v2`. It gives you detailed installation instructions for Flux v2.
```

Comment:
```suggestion
I highly recommended that you check out the [getting started](https://fluxcd.io/docs/get-started/) page of `Flux v2`. It gives you detailed installation instructions for Flux v2.
```

Reply:
thank you so much for fixing the issues, it should be fixed now 🙏